app/include/user_version.h

```diff
@@ -11,10 +11,10 @@
 #define NODE_VERSION_STR(x)	#x
 #define NODE_VERSION_XSTR(x)	NODE_VERSION_STR(x)
 
-#define NODE_VERSION		"NodeMCU " ESP_SDK_VERSION_STRING "." NODE_VERSION_XSTR(NODE_VERSION_INTERNAL)
```

Comment:
The changes in this file need to be reverted.

Reply:
Ups. done now.